Apps/WebClient/src/ClientApp/src/components/calendar/header.vue

```diff
@@ -78,26 +80,21 @@ export default class CalendarComponent extends Vue {
 
     private monthIndex: number = 0;
     private headerDate: Date = new Date();
+    private eventBus = EventBus;
     private leftIcon: string = "chevron-left";
     private rightIcon: string = "chevron-right";
 
     public get title(): string {
         return moment(this.currentDate).format(this.titleFormat);
     }
 
-    @Watch("availableMonths")
```

Comment:
without this, the navigation might move to months that don't have data. Can you make sure the month toggle only moves to months that have data?

Reply:
auto-merge issue, I will need to revert the code back